yt_dlp/extractor/rai.py

```diff
@@ -166,6 +167,14 @@ def get_format_info(tbr):
                 'fps': 25,
             }
 
+        # Check if MP4 download is available
```

Comment:
Isn't this what `IE._check_formats()`/`IE._is_valid_url()` are for?

Reply:
Also, do I understand correctly from the comment in the issue that a failing mp4 media link could be transformed to become valid after removing a critical URL query parameter?